protoc-gen-orion/internal/generator/generator.go

```diff
@@ -1,26 +1,13 @@
-// protoc-gen-orion is a plugin for the Google protocol buffer compiler to generate
-// Orion Go code.  Run it by building this program and putting it in your path with
-// the name
-// 	protoc-gen-orion
-//
-// The generated code is documented in the package comment for
-// the library.
-package main
```

Comment:
keep the main function in protoc-gen-orion/main.go. And move the generating logic into internal/generator/generator.go file.